docs/development/build-instructions-linux.md

```diff
@@ -25,7 +25,7 @@ Follow the guidelines below for building Electron on Linux.
   Doing so permits installing Node on your own home directory as a standard user.
   Or try repositories such as [NodeSource](https://nodesource.com/blog/nodejs-v012-iojs-and-the-nodesource-linux-repositories).
 * [clang](https://clang.llvm.org/get_started.html) 3.4 or later.
-* Development headers of GTK+ and libnotify.
+* Development headers of GTK and libnotify.
```

Comment:
Nit:

```suggestion
* Development headers of GTK 3 and libnotify.
```

Charles, feel free to disagree as I defer to your judgment 😄

Reply:
My rationale was that GTK 3 came out in 2011, but you're right there are still some tools based on GTK 2 still kicking around out there. :+1: